### ISSUE TITLE
zappa.gun.io SSL Error

### ISSUE THREAD

**Issue by Miserlou** [Miserlou/Zappa#176]
Has it really been 3 months already? Need to re-issue a new cert, or, even better, make it auto-renewing now that scheduled functions work.

**Comment by brettlangdon** [comment]
@Miserlou hope you don't mind, but just wanted to ping you here since the expired SSL cert is still an issue.

**Comment by Miserlou** [comment]
Yeah, I'm sort of working on this right now. Rather than just manually do it again, I'm trying to make Zappa automatically renew certs..

**Comment by brettlangdon** [comment]
Awesome :+1:

**Comment by Miserlou** [comment]
Okay, so it turns out that proof-of-possession has been removed from the ACME spec and Boulder doesn't support it anyway, so I don't know. New plan needed.
